tests/queries/0_stateless/01756_optimize_skip_unused_shards_rewrite_in.sql

```diff
@@ -0,0 +1,127 @@
+-- NOTE: this test cannot use 'current_database = currentDatabase()',
```

Comment:
Interesting, this PR does not adds this test to the arcadia skip list, but I guess it had been passed eventually?